setup.py

```diff
@@ -93,7 +93,7 @@ def run(self):
           'google-auth-oauthlib',
           'requests>=2.23.0'
       ],
-      extra_require={'profiler' : ['memory_profiler'], 'LHCb' : ['LbDevTools']},
+      extras_require={'profiler' : ['memory_profiler'], 'LHCb' : ['LbDevTools']},
```

Comment:
Is this a completely unrelated bug fix in the dependencies?

Reply:
Yes it's just a bugfix for a typo. I noticed as doing `pip install ".[LHCb]"` would say "no extra called LHCb": https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords

Reply:
Not sure how this has worked for the LHCb installations for the past few months ...

Reply:
We haven't used the `LbDev` stuff or the `memory_profiler` in any releases. The `LbDev` stuff was added for future use with the new application environment - I am not sure what the status of that package is. Currently it is a bit of a hack by source LbEnv from CVMFS every time we do something.